tools/cvm-image-rewriter/plugins/05-readonly-data/pre-stage/host_run.sh

```diff
@@ -3,7 +3,7 @@
 DIR=$(dirname "$(readlink -f "$0")")
 FILE_LIST="$DIR/file_list"
 CLD_SH_READONLY_FILE="01-file-readonly.sh"
-CLD_SH="$DIR/cloud-init/x-shellscript/$CLD_SH_READONLY_FILE"
+CLD_SH="$DIR/../cloud-init/x-shellscript/$CLD_SH_READONLY_FILE"
```

Comment:
I tried to comment on the readme.md but it wont let me because only the name was changed.  My comment is how to users know how to enable this or not?  Is it that just by adding files to this then this plugin runs?

Reply:
Yes, unless you have added a `NOT_RUN` file in this plugin directory following the `tools/cvm-image-rewriter/README.md ` instructions.